src/main/java/fdv/controller/TicketController.java

```diff
@@ -0,0 +1,106 @@
+package fdv.controller;
+
+import fdv.entity.LoginData;
+import fdv.entity.PlaceStatus;
+import fdv.entity.Ticket;
+import fdv.service.*;
+import jakarta.servlet.http.HttpSession;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import java.util.List;
+
+
+@Controller
+@AllArgsConstructor
+@SessionAttributes({"loggedUser"})
+public class TicketController {
+
+    SeanceService seanceService;
+    FilmService filmService;
+    TicketService ticketService;
+    PlaceService placeService;
+    PlaceKindService placeKindService;
+    ServiceService serviceService;
+    TicketServiceService ticketServiceService;
```

Comment:
All statless variables which will be injected by Spring should have `privite final` access modifiers